bookwyrm/templates/lists/list.html

```diff
@@ -40,6 +81,16 @@
                             <input type="hidden" name="item" value="{{ item.id }}">
                             <button type="submit" class="button is-small is-danger">{% trans "Remove" %}</button>
                         </form>
+                    </div>
+                    <div class="card-footer has-background-white-bis">
+                        <div>
+                            <form name="set-position" method="post" action="{% url 'list-set-book-position' item.id %}" class="card-footer-item">
+                                {% csrf_token %}
+                                <label for="input-list-position" class="is-sr-only">{% trans "Set list position" %}</label>
+                                <input id="input-list-position" class="input" type="text" name="position" value="">
```

Comment:
It would also be helpful to have the existing position as the default value, especially since the order the items are displayed in may not correspond to the order indicated here

Reply:
This is a good idea. Changed type to number and set the minimum to 1.